### PR TITLE
[KotlinCleanup] NavigationDrawerActivity

### PR DESCRIPTION
**Pull request by n1snt**
## How Has This Been Tested?
- Tested on a physical device
- Tests

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.kt [hunk 1]
```diff
@@ -110,15 +110,16 @@ abstract class NavigationDrawerActivity :
...
         window.statusBarColor = ContextCompat.getColor(this, R.color.transparent)
-        mDrawerLayout!!.setStatusBarBackgroundColor(Themes.getColorFromAttr(this, R.attr.colorPrimary))
+        mDrawerLayout.setStatusBarBackgroundColor(Themes.getColorFromAttr(this, R.attr.colorPrimary))
         // Setup toolbar and hamburger
-        mNavigationView = mDrawerLayout!!.findViewById(R.id.navdrawer_items_container)
+        mNavigationView = mDrawerLayout.findViewById(R.id.navdrawer_items_container)
         mNavigationView!!.setNavigationItemSelectedListener(this)
         val toolbar: Toolbar? = mainView.findViewById(R.id.toolbar)
         if (toolbar != null) {
+            toolbar.isEnabled = true
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.kt:121]: Why was this added?

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.kt (hunk 1 above)
Comment (n1snt) [reply AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.kt:121]: I think I may have confused `.isEnabled` with `.apply` from the KotlinCleanup on line 107.

**Comment by n1snt** [comment]
@lukstbit Made the requested changes.

**Review by lukstbit — APPROVED** [review]
Thanks!

**Review by david-allison — APPROVED** [review]

**Pull request merged**